src/pages/blog/starlingx-release-4.md

```diff
@@ -0,0 +1,35 @@
+---
+templateKey: blog-post
+title: StarlingX 4.0 is here!
+author: Ildiko Vancsa
+date: 2020-08-03T01:32:05.627Z
+category: 
+  - label: News & Announcements
+    id: category-A7fnZYrE1
+---
+
+The StarlingX community has just launched the fourth release of the project with various new features and enhancements to the platform.<!-- more -->
+
+The project is approaching a half-year release cycle and this article will give you some insights about what the contributors were working on during the first half of this year.
+
+As the StarlingX platform is fine tuned for edge computing and IoT use cases there are several requirements it has to fulfill in areas such as performance, robustness and security which were in focus during the 4.0 release cycle as well. This blog post will give you insights to some of the new features and components that got integrated into the platform.
+
+Security and proper isolation are crucial in an edge environment as the hardware and software components are located outside of the safety of traditional data centers. This can put challenges on traditional container runtimes as they are sharing the host kernel and use namespaces for separation which can be problematic in case one of the containers gets compromised. There are multiple solutions to this one of which is to use [Kata Containers](https://katacontainers.io) which is a container runtime that provides a fusion between virtual machines and containers. You can now use this project in StarlingX as a runtime that is run by Kubernetes.
```

Comment:
Security and proper isolation are crucial in an edge environment since the hardware and software components are located outside of the safety of traditional data centers. This can put challenges on traditional container runtimes, as they share the host kernel and use namespaces for separation, which can be problematic if one of the containers is compromised. There are multiple solutions to this, one of which is to use [Kata Containers](https://katacontainers.io), a container runtime that provides a fusion between virtual machines and containers. You can now use this project in StarlingX as a runtime that is run by Kubernetes.